app/models/calendar.rb

```diff
@@ -1,5 +1,5 @@
 class Calendar
-  REPOSITORY_PATH = Rails.env.test? ? "test/fixtures" : "lib/data"
+  REPOSITORY_PATH = Rails.env.test? ? "spec/fixtures" : "lib/data"
```

Comment:
Oh no. I didn't realise this clause was in the model 😬.  A follow-up task should be to stub `REPOSITORY_PATH` in the tests. 😬